docs/content/docs/getting-started/build.md

```diff
@@ -175,8 +175,9 @@ curl http://localhost:8080/v1/chat/completions -H "Content-Type: application/jso
 
 #### Troublshooting mac
 
-1. If you encounter errors regarding a missing utility metal, install `Xcode` from the App Store.
-2. After the installation of Xcode, if you receive a xcrun error `'xcrun: error: unable to find utility "metal", not a developer tool or in PATH'`. You might have installed the Xcode command line tools before installing Xcode, the former one is pointing to an incomplete SDK.
+If you encounter errors regarding a missing utility metal, install `Xcode` from the App Store.
```

Comment:
removing the ordered list is a good change as these aren't steps to follow in order, but rather a series of things to check individually, but rather than removing the list layout entirely, lets change it to an unordered list, which you can do by just prepending each line with "- "

Reply:
@cryptk thanks for the suggestion! I've made anew commit to address it.